packages/template/src/screens/carousels.tsx

```diff
@@ -18,18 +20,22 @@ const ScreenCarousels = ({ navigation }: { navigation?: any }) => {
 
     return (
         <Screen style={[theme.styles.screen, styles.screen]}>
-            <List
-                items={data}
-                itemDimensions={{ height: isFactorMobile ? 200 : 250 }}
-                rowHeight={isFactorMobile ? 350 : 400}
-                animatorOptions={{ type: 'scale_with_border', scale: 1.1 }}
-                focusOptions={{ forbiddenFocusDirections: ['right'] }}
-                itemSpacing={isFactorMobile ? 15 : 30}
-                cardStyle={styles.cardStyle}
-                onPress={(data) => {
-                    navigation.navigate(ROUTES.DETAILS, { row: data.rowNumber, index: data.index });
-                }}
-            />
+            <TouchableOpacity
```

Comment:
Not good to have TouchableOpacity here. Can it be changed with at least <View />?

Reply:
It can be changed to anything what supports test id's, it does not have to be clickable or anything, just visible. What should I change it to?

Reply:
yes change it to View